package.json

```diff
@@ -33,6 +37,7 @@
     "@well-known-components/logger": "^3.0.0",
     "@well-known-components/metrics": "^2.0.1-20220909150423.commit-8f7e5bc",
     "aws-sdk": "^2.1311.0",
+    "decentraland-crypto-middleware": "^1.0.4",
```

Comment:
This one needs to be a regular dependency (not dev) a it's used in `resize-handler`. Regarding the others... I think they will also need to be prod dependencies as they are required for the middleware module to initialize.

Reply:
Will do, thank you!